src/CMakeLists.txt

```diff
@@ -12,9 +12,6 @@
 # the public, perform publicly and display publicly, and to permit others to do so.
 #=========================================================================================
 
-# Configure defs.hpp
-set(PROBLEM_GENERATOR "<not-implemented>") # TODO: Figure out what to put here
-
```

Comment:
I'm not sure I would delete this. I think this is still an outstanding problem we will eventually need to solve.